README.md

```diff
@@ -1,29 +1,59 @@
-# fictional-octo-doodle
-fictional-octo-doodle
+# Intro
 
-This project automates the deployment of OpenStack Helm on CAASP and
+This project automates the deployment of OpenStack-Helm on CaaSP and
 SES via a series of shell scripts and Ansible playbooks.
 
+# General requirements
+
+## Ansible
+
+You need to install Ansible 2.7 or later to run these playbooks.
```

Comment:
I think installing ansible 2.7 is error prone on opensuse system. Just now tried via pip install and cannot install 2.7.2 version.
May be we do this automatically (via pre-flight checks) if version is less than 2.7.x ( we need this for ansible reboot module) ? What do you think?

Reply:
I'd be curious to see what problem you were having with it. I've successfully installed and used 2.7.1 and just pip installed 2.7.2.